lib/logstash/inputs/patch.rb

```diff
@@ -0,0 +1,44 @@
+if Gem.loaded_specs['elasticsearch-transport'].version >= Gem::Version.new("7.2.0")
```

Comment:
let's make it clear here that this was fixed already in 7.2.0, but we're using 5.x, and also let's point to the PR where it was solved